pkg/webhook/state_into_spec_annotation_validator.go

```diff
@@ -51,13 +51,28 @@ func (a *stateIntoSpecAnnotationValidator) Handle(ctx context.Context, req admis
 	}
 
 	value, ok := k8s.GetAnnotation(k8s.StateIntoSpecAnnotation, obj)
+	klog.Infof("maqiuyu...annotation value %v, ok? %v", value, ok)
```

Comment:
nit: remove the debug message?

Reply:
Thanks, @gemmahou ! Will clean up if we decide we'll move forward with this change.